src/elements/content-explorer/Content.js

```diff
@@ -49,62 +50,73 @@ type Props = {
 };
 
 const Content = ({
-    view,
-    isSmall,
-    isMedium,
-    isTouch,
-    rootId,
-    rootElement,
-    currentCollection,
-    tableRef,
-    focusedRow,
-    canDownload,
     canDelete,
+    canDownload,
+    canPreview,
     canRename,
     canShare,
-    canPreview,
+    currentCollection,
+    focusedRow,
+    isGridView = false,
+    isMedium,
+    isSmall,
+    isTouch,
     onItemClick,
-    onItemSelect,
     onItemDelete,
     onItemDownload,
+    onItemPreview,
     onItemRename,
+    onItemSelect,
     onItemShare,
-    onItemPreview,
     onSortChange,
-}: Props) => (
-    <div className="bce-content">
-        {view === VIEW_ERROR || view === VIEW_SELECTED ? null : (
-            <ProgressBar percent={currentCollection.percentLoaded} />
-        )}
-        {isEmpty(view, currentCollection) ? (
-            <EmptyState view={view} isLoading={currentCollection.percentLoaded !== 100} />
-        ) : (
-            <ItemList
-                view={view}
-                isSmall={isSmall}
-                isMedium={isMedium}
-                isTouch={isTouch}
-                rootId={rootId}
-                rootElement={rootElement}
-                focusedRow={focusedRow}
-                currentCollection={currentCollection}
-                tableRef={tableRef}
-                canShare={canShare}
-                canPreview={canPreview}
-                canDelete={canDelete}
-                canRename={canRename}
-                canDownload={canDownload}
-                onItemClick={onItemClick}
-                onItemSelect={onItemSelect}
-                onItemDelete={onItemDelete}
-                onItemDownload={onItemDownload}
-                onItemRename={onItemRename}
-                onItemShare={onItemShare}
-                onItemPreview={onItemPreview}
-                onSortChange={onSortChange}
-            />
-        )}
-    </div>
-);
+    rootElement,
+    rootId,
+    tableRef,
+    view,
+}: Props) => {
+    const listView = (
```

Comment:
We should try to avoid rendering non-dynamic components into variables, when possible. If a render method is getting too complicated, it should probably be broken into sub-components.